pkg/registry/service/registry_test.go

```diff
@@ -0,0 +1,335 @@
+/*
```

Comment:
No. I will remove it. It's removed by #13584, I rebased mine on it, and git didn't warn me about adding it back (or maybe I missed it..).